sdk/eventhub/eventhubs-checkpointstore-blob/samples/v1-beta/javascript/receiveEventsWithApiSpecificStorage.js

```diff
@@ -15,25 +15,25 @@
 const { DefaultAzureCredential } = require("@azure/identity");
 const { EventHubConsumerClient } = require("@azure/event-hubs");
 const { BlobCheckpointStore } = require("@azure/eventhubs-checkpointstore-blob");
-const { ContainerClient } = require("@azure/storage-blob");
-const { createCustomPipeline } = require("./createCustomPipeline");
+const { BlobServiceClient } = require("@azure/storage-blob");
+const { createCustomPipeline } = require("./createCustomPipeline.js");
```

Comment:
Does the javascript/typescript samples' package.json files need to have `"type": "module"` now? They are supposed to be standalone and published to Microsoft Samples sites.

Reply:
no, just tried both js and ts samples and they work fine.